app/c_simulation.py

```diff
@@ -10,13 +10,15 @@ def unique(self):
     @property
     def display(self):
         return dict(name=self.name, ability_id=self.ability_id, paw=self.paw, status=self.status,
-                    response=self.response)
+                    response=self.response, variable_name=self.variable_name, variable_value=self.variable_value)
 
-    def __init__(self, name, ability_id, paw, status, response):
+    def __init__(self, name, ability_id, paw, status, response, variable_name=None, variable_value=None):
         self.name = name
         self.ability_id = ability_id
         self.paw = paw
         self.status = status
+        self.variable_name = variable_name
```

Comment:
I couldn't figure out why this was confusing me and just did... we should be consistent with the Fact() object and other references throughout the application with regards to facts:
`self.variable_trait = variable_trait`

Reply:
Updated it here and where applicable elsewhere.